misc/scripts/vfile-swarm-setup.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#source ../misc/scripts/commands.sh
```

Comment:
nit: can this be removed?

Reply:
I need $SSH which defined in commands.sh

Reply:
isn't this dead code (commented out)? I thought L#17 is sufficient.

Reply:
Fixed.